### ISSUE TITLE
Bug: Mis-expectation of indentation after a multiline backtick string

### ISSUE THREAD

**Issue by eyalroz** [eslint/eslint#17507]
### Environment

I'll first say that your code of conduct is horrible, oppressive, unjust, and befits a bleak dystopia rather than a free software project. You should really get rid of it.

Node version: 18.13.0
npm version: 9.2.0
Local ESLint version: v8.45.0
Global ESLint version: v8.45.0 (not sure what the difference is - I only use one version)
Operating System: Devuan GNU/Linux Excalibur (like Debian Trixie but with no systemd)

### What parser are you using?

Default (Espree)

### What did you do?

<details>
<summary>Configuration</summary>

```json
{
  "env": {
    "browser": true,
    "es6": true,
    "es2021": true
  },
  "extends": [
    "airbnb-base"
  ],
  "parserOptions": {
    "ecmaVersion": 2022
  },
  "rules": {
    "indent": [
      "error",
      2,
      {
        "SwitchCase": 0,
        "MemberExpression": "off",
        "ignoreComments": true
      }
    ],
    "function-paren-newline": "off",
    "no-unused-vars": "off"
  }
}
```
</details>

```js
const foo = 0;
function bar(str, i) { return str.length + i; }

function baz() {
  bar(`
some text here
and more text here
`,
    foo);
}
```

### What did you expect to happen?

No warnings/errors

### What actually happened?

Got:
```
/path/to/file.js
  9:1  error  Expected indentation of 2 spaces but found 4  indent
```

### Link to Minimal Reproducible Example

No need, code above is sufficient

### Participation

- [ ] I am willing to submit a pull request for this issue.

### Additional comments

The backticked string is actually really really long and can absolutely not fit on one line (or 2, or 10).

**Comment by mdjermanovic** [comment]
This might be related to https://github.com/eslint/eslint/issues/17013. Indentation of arguments seems different when the first argument is multiline and starts on the same line as `(`.

**Comment by eyalroz** [comment]
@mdjermanovic : Sounds plausible, but it's not obvious that the relation is material rather just "symptomatic".

**Comment by eyalroz** [comment]
Don't auto-close the issue.

**Comment by nzakas** [comment]
Hi folks, I just wanted to let everyone know that the ESLint team has decided to deprecate all formatting rules, including indent, and recommend people use source code formatters like [Prettier](https://prettier.io) or [dprint](https://dprint.dev) instead. As a result, we are closing all outstanding issues related to these rules. For more information, please see https://github.com/eslint/eslint/issues/17522. Thanks.
